lib/gui/app/models/store.js

```diff
@@ -499,6 +503,16 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
       return state.deleteIn([ 'selection', 'image' ])
     }
 
+    case ACTIONS.SET_APPLICATION_SESSION_UUID: {
+      return state
+        .set('applicationSessionUuid', action.data)
```

Comment:
Why is there a new line here?

Reply:
There are newlines everywhere before each new `case`, so I figured I'd keep them in the added cases too

Reply:
I mean

```js
return state
        .set('applicationSessionUuid', action.data)
```

There's no chaining going on here, so no point in sticking an EOL in front of a method call ;)